src/logion/migration/1638955476613-AddAttributesForDraft.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class AddAttributesForDraft1638955476613 implements MigrationInterface {
+    name = 'AddAttributesForDraft1638955476613'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "public"."loc_request_file" ADD "nature" character varying(255)`);
```

Comment:
It would be nice to add the same length constraint on nature front-side.

Reply:
Done.
See https://github.com/logion-network/logion-wallet/blob/main/src/legal-officer/transaction-protection/LocPrivateFileForm.tsx#L56
and also https://github.com/logion-network/logion-wallet/blob/main/src/legal-officer/transaction-protection/LocPrivateFileForm.tsx#L35